### PR TITLE
25.0 release notes: batch 2

### PR DESCRIPTION
**Pull request by ektravel**
Second batch of draft release notes for 25.0 for review.

This PR has:

- [x] been self-reviewed

### CODE REVIEW

File: docs/next-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+
+The MSQ task engine now supports the `indexSpec` context parameter.
+
+https://github.com/apache/druid/pull/13275
+
+### Added task start status to the worker report
+
+Added `pendingTasks` and `runningTasks` fields to the worker report for the MSQ task engine.
+
+https://github.com/apache/druid/pull/13263
```
Comment (vogievetsky) [inline docs/next-release-notes.md:51]: This should also mention https://github.com/apache/druid/pull/13291 which is the UI followup to the same work.

Comment (vogievetsky) [reply docs/next-release-notes.md:51]: Although I guess it is mentioned in the web console section... maybe these sections should link to each other :-p

**Comment by 317brian** [comment]
I branched off this, so you can close this one @ektravel

**Comment by kfaraz** [comment]
Closing this as Druid 25.0 is released.

**Pull request closed without merging**